locales/en-US/server.ftl

```diff
@@ -81,6 +81,8 @@ notFoundPageDescription = Page not found.
 
 [[ Shot page ]]
 
+# This is the page title, put into the page <title></title>
```

Comment:
Wrong format for the comment. Make sure to run compare-locales locally ;-)

Also, I would simplify it: "This is used for the HTML title tag of the page